modules/yieldNexusBidAdapter.js

```diff
@@ -0,0 +1,157 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { config } from 'src/config';
+
+const pixKey = 'utrk';
+
+function startsWith(str, search) {
+  return str.substr(0, search.length) === search;
+}
+
+export const spec = {
+  code: 'yieldnexus',
+  aliases: [],
+  supportedMediaTypes: ['banner', 'video'],
+
+  isBidRequestValid: function(bid) {
+    if (!bid.params.spid) {
+      return false;
+    } else if (typeof bid.params.spid !== 'string') {
+      return false;
+    }
+    return (typeof bid.params.instl === 'undefined' || bid.params.instl === 0 || bid.params.instl === 1) &&
+           (typeof bid.params.bidfloor === 'undefined' || typeof bid.params.bidfloor === 'number') &&
+           (typeof bid.params['protocols'] === 'undefined' || Array.isArray(bid.params['protocols'])) &&
+           (typeof bid.params['adpos'] === 'undefined' || typeof bid.params['adpos'] === 'number');
+  },
+
+  buildRequests: function(validBidRequests, bidderRequest) {
+    return validBidRequests.map(bidRequest => {
+      let referrer = '';
+      try {
+        referrer = window.top.document.referrer;
+      } catch (e) {
+        try {
+          referrer = window.document.referrer;
+        } catch (e) {
+        }
+      }
+      const url = utils.getTopWindowUrl();
+      const domainStart = url.indexOf('://') + 3;
+      const req = {
+        id: bidRequest.auctionId,
+        site: {
+          domain: url.substring(domainStart, url.indexOf('/', domainStart) < 0 ? url.length : url.indexOf('/', domainStart)),
+          page: config.getConfig('pageUrl') || utils.getTopWindowUrl(),
```

Comment:
I don't believe `config.getConfig('pageUrl')` will ever return anything; so it could potentially be removed.

Additionally, it seems like you already obtain the result from `utils.getTopWindowUrl()` for the `url` variable a little above.  Could you reuse that variable here for the `page` parameter?